tests/common/fixtures/advanced_reboot.py

```diff
@@ -756,7 +756,8 @@ def __runPtfRunner(self, rebootOper=None):
             params=params,
             log_file='/tmp/advanced-reboot.ReloadTest.log',
             module_ignore_errors=self.moduleIgnoreErrors,
-            timeout=REBOOT_CASE_TIMEOUT
+            timeout=REBOOT_CASE_TIMEOUT,
+            is_python3=True
```

Comment:
Where is this `is_python3` argument being used?

Reply:
In ptf_runner.py, if is_python3==true, Python3 is used to run the PTF script.